host-configs/hip.cmake

```diff
@@ -0,0 +1,21 @@
+##############################################################################
+# Copyright (c) 2018-2019, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory
+#
+# Created by David Beckingsale, david@llnl.gov
+# LLNL-CODE-747640
+#
+# All rights reserved.
+#
+# This file is part of Umpire.
+#
+# For details, see https://github.com/LLNL/Umpire
+# Please also see the LICENSE file for MIT license.
+##############################################################################
+
+set(ENABLE_HIP ON CACHE BOOL "")
+set(ENABLE_OPENMP OFF CACHE BOOL "")
+set(ENABLE_CUDA Off CACHE BOOL "")
+
+set(HIP_HIPCC_FLAGS "--amdgpu-target=gfx900" CACHE STRING "")
+set(HIP_ROOT_DIR "/opt/rocm/hip" CACHE PATH "HIP ROOT directory path")
```

Comment:
Should the root directory be hard-coded like this?  Should a warning be generated if the root directory cannot be found?

Reply:
It's an option that can be overridden, and something like that is already in the develop branch in `rocm.cmake`. Once there are machines, they can be switched to correct paths.

Reply:
Yeah this a config that works at LLNL. If you don't supply this file to cmake (using the `-C` option), then the FindHIP module will look in the default locations to try and find everything.